portal/app/[locale]/_components/navbar/_components/getStarted.tsx

```diff
@@ -29,7 +29,7 @@ export const GetStarted = function () {
         onClick={track ? onClick : undefined}
       >
         <span>{t('get-started')}</span>
-        <Chevron.Right />
+        <Chevron.Right className="[&>path]:fill-white " />
```

Comment:
Minor, but there's a small space at the end that can be removed

```suggestion
        <Chevron.Right className="[&>path]:fill-white" />
```

Reply:
fixed